test/lib.url.spec.ts

```diff
@@ -3,46 +3,39 @@
 
 import { format, isAbsolute, combine, concatParams } from '../src/url';
 
-
 describe('URL Handler', () => {
-
   describe('concatParams', () => {
     it('stringify and concats params to the provided URL', () => {
-      const url    = 'https://www.foo.com/bar';
-      const params =  {
-        foo: {
-          bar: {
-            baz: 'foobarbaz'
-          }
-        }
-      };
-      const actual = concatParams(url, params)
-      const expectted = 'https://www.foo.com/bar?foo%5Bbar%5D%5Bbaz%5D=foobarbaz'
+      const url = 'https://www.foo.com/bar';
+      const params = { bar: 'bar', foo: 'foo' };
+      const actual = concatParams(url, params);
+      const expectted = 'https://www.foo.com/bar?bar=bar&foo=foo';
+      const url = 'https://www.foo.com/bar';
```

Comment:
This `url` is already defined at line 9 😇

Reply:
```suggestion
```